### ISSUE TITLE
nr_cluster.tsv with taxonomy ID for each of the members

### ISSUE THREAD

**Issue by RocesV** [Arcadia-Science/2023-nr-clustering#8]
Dear @mertcelebi @taylorreiter,

first of all thank you for advancing this useful resource! May be i am missing this but could be possible to include in the nr_cluster.tsv file the taxonomy ID (taxID) for each of members of the cluster? This would be helpful. Example:

`WP_206332294.1          WP_206332294.1       1883`
`WP_206332294.1           WP_099014929.1       92644`
`...`
Again, thank you very much in advance! 😄

**Comment by taylorreiter** [comment]
Hi @RocesV! That's a good suggestion, we'll look in to adding this info. I think this rule in the snakefile may generate an intermediate that might be useful for this: https://github.com/Arcadia-Science/2023-nr-clustering/blob/main/Snakefile#L84

I'm in the middle of wrapping up a different project at the moment so it may take me a couple weeks to get to this.

Just out of curiosity and if you're willing to share, what would you be using this for? Knowing this might help us think if there's a better way to present the information.

**Comment by RocesV** [comment]
Thank you for the very fast response @taylorreiter!

This is a side project still without a clear idea, so i can wait a couple weeks without problem and looking forward to use the resource! 😃

**Comment by taylorreiter** [comment]
Hi @RocesV, I found some time to return to this! Would a format like this work for you?

```
CCA64214.1	223311;10827;10827;10827;10827;10827;10827;223308;10827;1227357;408719
YP_009506371.1	223316;223316;327280;327280;327280;1312723;403897;408719
QDH06051.1	348800;392749;929769;1093958;1513525;1093958;889203;408719
YP_794166.1	1890423;1513525;649996;403897;408719
```
This has the representative sequence for each cluster along with all of the observed taxids for sequences that landed in that cluster. 

Unfortunately, I deleted many of the intermediate files that I would need to quickly produce the exact format you requested, which means it would somewhat expensive to create the exact format you requested. If the one above would work, let me know and I can make it available for download!
